2024-10-08 18:23:00,621 INFO    StreamThr :9812 [internal.py:wandb_internal():85] W&B internal server running at pid: 9812, started at: 2024-10-08 18:23:00.620317
2024-10-08 18:23:00,630 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: status
2024-10-08 18:23:00,634 INFO    WriterThread:9812 [datastore.py:open_for_write():87] open: .\wandb\run-20241008_182300-qhai7aei\run-qhai7aei.wandb
2024-10-08 18:23:00,639 DEBUG   SenderThread:9812 [sender.py:send():379] send: header
2024-10-08 18:23:00,640 DEBUG   SenderThread:9812 [sender.py:send():379] send: run
2024-10-08 18:23:00,647 INFO    SenderThread:9812 [sender.py:_setup_resume():749] checking resume status for None/rl4co/qhai7aei
2024-10-08 18:23:01,863 INFO    SenderThread:9812 [dir_watcher.py:__init__():211] watching files in: .\wandb\run-20241008_182300-qhai7aei\files
2024-10-08 18:23:01,863 INFO    SenderThread:9812 [sender.py:_start_run_threads():1188] run started: qhai7aei with start time 1728382980.622358
2024-10-08 18:23:01,885 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: check_version
2024-10-08 18:23:01,886 DEBUG   SenderThread:9812 [sender.py:send_request():406] send_request: check_version
2024-10-08 18:23:03,190 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: run_start
2024-10-08 18:23:04,425 DEBUG   HandlerThread:9812 [system_info.py:__init__():26] System info init
2024-10-08 18:23:04,426 DEBUG   HandlerThread:9812 [system_info.py:__init__():41] System info init done
2024-10-08 18:23:04,426 INFO    HandlerThread:9812 [system_monitor.py:start():194] Starting system monitor
2024-10-08 18:23:04,428 INFO    HandlerThread:9812 [system_monitor.py:probe():214] Collecting system info
2024-10-08 18:23:04,428 INFO    SystemMonitor:9812 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-10-08 18:23:04,430 INFO    SystemMonitor:9812 [interfaces.py:start():188] Started cpu monitoring
2024-10-08 18:23:04,436 INFO    SystemMonitor:9812 [interfaces.py:start():188] Started disk monitoring
2024-10-08 18:23:04,439 INFO    SystemMonitor:9812 [interfaces.py:start():188] Started gpu monitoring
2024-10-08 18:23:04,441 INFO    SystemMonitor:9812 [interfaces.py:start():188] Started memory monitoring
2024-10-08 18:23:04,444 INFO    SystemMonitor:9812 [interfaces.py:start():188] Started network monitoring
2024-10-08 18:23:04,461 DEBUG   HandlerThread:9812 [system_info.py:probe():152] Probing system
2024-10-08 18:23:04,461 DEBUG   HandlerThread:9812 [system_info.py:probe():200] Probing system done
2024-10-08 18:23:04,461 DEBUG   HandlerThread:9812 [system_monitor.py:probe():223] {'os': 'Windows-10-10.0.22631-SP0', 'python': '3.9.15', 'heartbeatAt': '2024-10-08T10:23:04.461834', 'startedAt': '2024-10-08T10:23:00.490701', 'docker': None, 'cuda': None, 'args': (), 'state': 'running', 'program': 'E:\\rl4ev\\train.py', 'codePathLocal': 'train.py', 'codePath': 'train.py', 'host': 'DESKTOP-4UKOA1U', 'username': '78788', 'executable': 'd:\\py\\Anaconda3\\envs\\pytorch\\python.exe', 'cpu_count': 6, 'cpu_count_logical': 6, 'cpu_freq': {'current': 2901.0, 'min': 0.0, 'max': 2901.0}, 'cpu_freq_per_core': [{'current': 2901.0, 'min': 0.0, 'max': 2901.0}], 'disk': {'/': {'total': 953.8369140625, 'used': 35.10498046875}}, 'gpu': 'NVIDIA GeForce GTX 1050 Ti', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce GTX 1050 Ti', 'memory_total': 4294967296}], 'memory': {'total': 7.857707977294922}}
2024-10-08 18:23:04,462 INFO    HandlerThread:9812 [system_monitor.py:probe():224] Finished collecting system info
2024-10-08 18:23:04,462 INFO    HandlerThread:9812 [system_monitor.py:probe():227] Publishing system info
2024-10-08 18:23:04,462 DEBUG   HandlerThread:9812 [system_info.py:_save_conda():209] Saving list of conda packages installed into the current environment
2024-10-08 18:23:04,474 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:04,474 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:04,903 INFO    Thread-17 :9812 [dir_watcher.py:_on_file_created():271] file/dir created: .\wandb\run-20241008_182300-qhai7aei\files\conda-environment.yaml
2024-10-08 18:23:06,907 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:06,908 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:08,921 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:08,922 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:11,338 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:11,339 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:13,754 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:13,754 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:16,163 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:16,163 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:18,580 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:18,581 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:19,489 ERROR   HandlerThread:9812 [system_info.py:_save_conda():223] Error saving conda packages: Command '['conda', 'env', 'export']' timed out after 15 seconds
Traceback (most recent call last):
  File "d:\py\Anaconda3\envs\pytorch\lib\site-packages\wandb\sdk\internal\system\system_info.py", line 216, in _save_conda
    subprocess.call(
  File "d:\py\Anaconda3\envs\pytorch\lib\subprocess.py", line 351, in call
    return p.wait(timeout=timeout)
  File "d:\py\Anaconda3\envs\pytorch\lib\subprocess.py", line 1189, in wait
    return self._wait(timeout=timeout)
  File "d:\py\Anaconda3\envs\pytorch\lib\subprocess.py", line 1473, in _wait
    raise TimeoutExpired(self.args, timeout)
subprocess.TimeoutExpired: Command '['conda', 'env', 'export']' timed out after 15 seconds
2024-10-08 18:23:19,497 DEBUG   HandlerThread:9812 [system_info.py:_save_conda():224] Saving conda packages done
2024-10-08 18:23:19,502 INFO    HandlerThread:9812 [system_monitor.py:probe():229] Finished publishing system info
2024-10-08 18:23:19,514 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 18:23:19,514 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: keepalive
2024-10-08 18:23:19,514 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 18:23:19,514 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: keepalive
2024-10-08 18:23:19,515 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 18:23:19,515 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: keepalive
2024-10-08 18:23:19,516 DEBUG   SenderThread:9812 [sender.py:send():379] send: files
2024-10-08 18:23:19,516 INFO    SenderThread:9812 [sender.py:_save_file():1454] saving file wandb-metadata.json with policy now
2024-10-08 18:23:20,008 INFO    Thread-17 :9812 [dir_watcher.py:_on_file_created():271] file/dir created: .\wandb\run-20241008_182300-qhai7aei\files\wandb-metadata.json
2024-10-08 18:23:20,351 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: python_packages
2024-10-08 18:23:20,351 DEBUG   SenderThread:9812 [sender.py:send_request():406] send_request: python_packages
2024-10-08 18:23:20,355 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: stop_status
2024-10-08 18:23:20,356 DEBUG   SenderThread:9812 [sender.py:send_request():406] send_request: stop_status
2024-10-08 18:23:20,365 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:23:20,719 DEBUG   SenderThread:9812 [sender.py:send():379] send: telemetry
2024-10-08 18:23:20,719 DEBUG   SenderThread:9812 [sender.py:send():379] send: telemetry
2024-10-08 18:23:20,720 DEBUG   SenderThread:9812 [sender.py:send():379] send: metric
2024-10-08 18:23:20,720 DEBUG   SenderThread:9812 [sender.py:send():379] send: telemetry
2024-10-08 18:23:20,721 DEBUG   SenderThread:9812 [sender.py:send():379] send: metric
2024-10-08 18:23:20,721 WARNING SenderThread:9812 [sender.py:send_metric():1405] Seen metric with glob (shouldn't happen)
2024-10-08 18:23:20,839 INFO    wandb-upload_0:9812 [upload_job.py:push():130] Uploaded file C:\Users\78788\AppData\Local\Temp\tmp3lto80ftwandb\s2cgkrkr-wandb-metadata.json
2024-10-08 18:23:21,024 INFO    Thread-17 :9812 [dir_watcher.py:_on_file_created():271] file/dir created: .\wandb\run-20241008_182300-qhai7aei\files\requirements.txt
2024-10-08 18:23:21,041 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:21,041 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:21,372 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:23:21,734 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 18:23:22,375 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:23:23,055 INFO    Thread-17 :9812 [dir_watcher.py:_on_file_modified():288] file/dir modified: .\wandb\run-20241008_182300-qhai7aei\files\conda-environment.yaml
2024-10-08 18:23:23,389 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:23:23,465 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:23,466 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:24,400 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:23:25,409 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:23:25,882 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:25,882 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:26,419 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:23:27,402 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 18:23:27,434 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:23:28,308 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:28,309 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:28,420 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:23:29,431 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:23:30,445 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:23:30,741 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:30,741 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:31,460 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:23:32,426 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 18:23:32,473 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:23:33,152 INFO    Thread-17 :9812 [dir_watcher.py:_on_file_modified():288] file/dir modified: .\wandb\run-20241008_182300-qhai7aei\files\config.yaml
2024-10-08 18:23:33,159 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:33,160 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:33,526 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:23:34,744 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:23:35,482 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: stop_status
2024-10-08 18:23:35,483 DEBUG   SenderThread:9812 [sender.py:send_request():406] send_request: stop_status
2024-10-08 18:23:35,572 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:35,572 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:35,750 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:23:36,780 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:23:37,766 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 18:23:37,799 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:23:37,982 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:37,983 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:38,813 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:23:39,225 INFO    Thread-17 :9812 [dir_watcher.py:_on_file_created():271] file/dir created: .\wandb\run-20241008_182300-qhai7aei\files\output.log
2024-10-08 18:23:39,828 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:23:40,428 ERROR   gpu       :9812 [interfaces.py:aggregate():159] Failed to serialize metric: division by zero
2024-10-08 18:23:40,438 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:40,439 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:40,838 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:23:41,852 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:23:42,407 DEBUG   SenderThread:9812 [sender.py:send():379] send: config
2024-10-08 18:23:42,864 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:23:42,874 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:42,875 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:43,414 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 18:23:43,874 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:23:44,886 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:23:44,897 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:44,897 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:45,882 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:23:46,888 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:23:46,915 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:46,915 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:47,897 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:23:48,461 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 18:23:48,911 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:23:48,937 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:48,937 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:49,927 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:23:50,399 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: stop_status
2024-10-08 18:23:50,400 DEBUG   SenderThread:9812 [sender.py:send_request():406] send_request: stop_status
2024-10-08 18:23:50,941 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:23:50,950 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:50,950 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:51,954 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:23:52,958 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:23:52,968 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:52,968 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:53,788 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 18:23:53,963 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:23:54,973 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:23:55,031 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:55,032 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:55,986 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:23:56,991 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:23:57,048 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:57,048 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:57,999 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:23:59,007 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:23:59,063 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:59,064 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:23:59,483 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 18:24:00,023 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:01,038 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:01,080 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:01,080 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:02,050 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:03,064 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:03,106 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:03,106 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:04,080 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:04,458 DEBUG   SystemMonitor:9812 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2024-10-08 18:24:04,460 DEBUG   SenderThread:9812 [sender.py:send():379] send: stats
2024-10-08 18:24:05,088 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:05,129 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:05,130 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:05,407 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: stop_status
2024-10-08 18:24:05,408 DEBUG   SenderThread:9812 [sender.py:send_request():406] send_request: stop_status
2024-10-08 18:24:05,747 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 18:24:06,094 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:06,455 INFO    Thread-17 :9812 [dir_watcher.py:_on_file_modified():288] file/dir modified: .\wandb\run-20241008_182300-qhai7aei\files\config.yaml
2024-10-08 18:24:07,105 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:07,542 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:07,542 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:08,114 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:09,121 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:09,556 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:09,556 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:10,127 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:11,465 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 18:24:11,586 ERROR   gpu       :9812 [interfaces.py:aggregate():159] Failed to serialize metric: division by zero
2024-10-08 18:24:11,596 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:11,596 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:11,721 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:12,724 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:13,609 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:13,609 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:13,742 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:14,741 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:15,621 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:15,621 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:15,755 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:16,482 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 18:24:16,768 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:17,644 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:17,644 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:17,778 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:18,781 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:19,796 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:20,073 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:20,073 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:20,406 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: stop_status
2024-10-08 18:24:20,407 DEBUG   SenderThread:9812 [sender.py:send_request():406] send_request: stop_status
2024-10-08 18:24:20,802 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:21,685 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 18:24:21,813 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:22,091 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:22,092 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:22,827 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:23,841 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:24,105 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:24,106 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:24,845 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:25,855 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:26,119 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:26,119 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:26,871 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:27,518 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 18:24:27,869 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:28,131 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:28,132 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:28,883 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:29,889 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:30,148 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:30,148 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:30,900 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:31,910 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:32,522 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 18:24:32,570 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:32,571 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:32,916 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:33,922 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:34,477 DEBUG   SenderThread:9812 [sender.py:send():379] send: stats
2024-10-08 18:24:34,597 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:34,597 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:34,934 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:35,453 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: stop_status
2024-10-08 18:24:35,454 DEBUG   SenderThread:9812 [sender.py:send_request():406] send_request: stop_status
2024-10-08 18:24:35,944 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:36,621 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:36,621 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:36,960 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:37,819 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 18:24:37,977 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:38,646 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:38,647 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:38,987 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:39,997 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:40,673 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:40,673 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:41,009 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:42,025 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:42,733 ERROR   gpu       :9812 [interfaces.py:aggregate():159] Failed to serialize metric: division by zero
2024-10-08 18:24:42,744 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:42,744 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:43,017 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:43,625 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 18:24:44,088 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:44,766 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:44,766 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:45,104 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:46,102 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:46,789 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:46,789 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:47,112 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:48,125 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:48,804 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:48,804 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:49,142 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:49,599 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 18:24:50,151 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:50,454 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: stop_status
2024-10-08 18:24:50,455 DEBUG   SenderThread:9812 [sender.py:send_request():406] send_request: stop_status
2024-10-08 18:24:50,831 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:50,831 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:51,155 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:52,161 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:52,846 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:52,846 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:53,165 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:54,181 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:54,800 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 18:24:54,886 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:54,887 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:55,191 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:56,193 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:56,909 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:56,909 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:57,197 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:58,199 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:24:58,931 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:58,932 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:24:59,188 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:25:00,192 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:25:00,602 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 18:25:00,966 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:25:00,966 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:25:01,207 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:25:02,213 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:25:02,995 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:25:02,995 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:25:03,214 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:25:04,208 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:25:04,492 DEBUG   SenderThread:9812 [sender.py:send():379] send: stats
2024-10-08 18:25:05,210 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:25:05,389 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:25:05,389 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:25:05,476 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: stop_status
2024-10-08 18:25:05,477 DEBUG   SenderThread:9812 [sender.py:send_request():406] send_request: stop_status
2024-10-08 18:25:05,765 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 18:25:06,216 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:25:07,221 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:25:07,401 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:25:07,401 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:25:08,226 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:25:09,235 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:25:09,441 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:25:09,442 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:25:10,242 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:25:11,248 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:25:11,463 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:25:11,463 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:25:11,610 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 18:25:12,262 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:25:13,258 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:25:13,512 ERROR   gpu       :9812 [interfaces.py:aggregate():159] Failed to serialize metric: division by zero
2024-10-08 18:25:13,523 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:25:13,523 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:25:14,264 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:25:15,254 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:25:15,548 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:25:15,548 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:25:16,262 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:25:16,613 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 18:25:17,269 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:25:17,565 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:25:17,565 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:25:18,581 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:25:19,595 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 18:25:19,601 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:25:19,602 ERROR   gpu       :9812 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 18:25:20,471 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: stop_status
2024-10-08 18:25:20,471 DEBUG   SenderThread:9812 [sender.py:send_request():406] send_request: stop_status
2024-10-08 18:25:20,612 DEBUG   HandlerThread:9812 [handler.py:handle_request():158] handle_request: internal_messages
